regulations/tasks.py

```diff
@@ -72,10 +74,16 @@ def submit_comment(self, body, files):
     except (ClientError, RequestException) as exc:
         logger.exception(exc)
         raise self.retry(exc=exc)
+    except Task.MaxRetriesExceededError as exc:
```

Comment:
That is likely. As per the docs, `self.retry(exc=exc)` should raise `exc` instead of `MaxRetriesExceededError` when max retries are exceeded. If it was indeed being raised inside `except (ClientError, RequestException)`, we would never get to `except MaxRetriesExceededError`.